### PR TITLE
Bugfix that enables rendering additionalProperties set to true.

### PR DESCRIPTION
**Pull request by m-mohr**
Solves issue #596.

### CODE REVIEW

File: src/services/models/Schema.ts [hunk 1]
```diff
@@ -244,7 +244,7 @@ function buildFields(
     sortByRequired(fields, schema.required);
   }
 
-  if (typeof additionalProps === 'object') {
+  if (typeof additionalProps === 'object' || typeof additionalProps === 'boolean') {
```
Comment (RomanGotsiy) [inline src/services/models/Schema.ts]: if `additionalProperties === false` you should not push "additional properties" field, change this to checking if it is implicit `true`:

```js
if (typeof additionalProps === 'object' || additionalProps === true) {
```

**Comment by m-mohr** [comment]
@RomanGotsiy Yes, that makes sense! Thanks for the review. Applied your suggestions, please re-check.

File: src/services/models/Schema.ts (hunk 1 above)
Comment (m-mohr) [reply src/services/models/Schema.ts]: Implemented.

**Comment by RomanGotsiy** [comment]
Nice 👍
Thanks!

**Pull request merged**